recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -1,5 +1,12 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
+# When we build for StarterKit with Kingficher, we use Cogent's device tree.
+# Cogent use their own name for ak4613 sound card - 'ak4613',
+# while on other boards it has name 'rcar-sound'.
+# This affects configuration file used by alsactl for setting of volume levels.
+# This is why we need dedicated config for SK+KF.
+FILESEXTRAPATHS_prepend_h3ulcb_4x2g_kf := "${THISDIR}/files/cog_kf:"
```

Comment:
I would use something less magic than cog_kf... kf is only cogent supported AFAIK, so s/cog_kf/kingfisher would sound better IMHO

Reply:
Done. Renamed to 'kf'.